### PR TITLE
Add `ignore: ["keyframe-selectors"]` to `rule-selector-property-disallowed-list`

### LINKED ISSUE #7511: Add `ignore: ["keyframe-selectors"]` to `rule-selector-property-disallowed-list`
### What minimal example or steps are needed to reproduce the bug?

```css
@keyframes fade-in {
  from {
    opacity: 0;
  }

  to {
    opacity: 1;
  }
}
```
### What minimal configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "rule-selector-property-disallowed-list": [{
        "/^[a-z]+$/": ["opacity"]
      }
    ]
  }
}
```

### How did you run Stylelint?

[Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIACBrGBPAZgJwIYFsYBnAAk11gFoBLAOxOAB16zsJ8HmTuSIAHXGGoAXdPBIAGANxcSAX2azhETi279BIsSQCMMlgtpyQAGhCZqAGxgA5AnEQwAHgT7WAdGCJFT4CLQsAcwQQJhZGEGwAV2siCPEwnhII6OtKIhhrMGVsSj42PhhsUUooaiJcS0sIAHcYKEpLcuF4kgBtRKSeCIB6AD023EoALwBdAGoAEh7WtoiNIVEI0dkkwySVg2ZjM0gA6kCAMQhsfFxhEIArIn9fWD4fRE6IolFrJtoWhGSQS3PiL4mWQvN6ZOjCSh7ILpYS4WhQXDYKCtCJ-YQAiLbEByIA)

### Which Stylelint-related dependencies are you using?

```json
{
  "stylelint": "latest",
  "stylelint-config-standard": "latest"
}
```

### What did you expect to happen?

No problems to be reported.

### What actually happened?

The following problem was reported: ```Unexpected property "opacity" for selector "from" [3:5-12]```

### Do you have a proposal to fix the bug?

Maybe the same fix as this ? #7417

### PR DESCRIPTION
**Pull request by emmacharp**
> Which issue, if any, is this issue related to?

Closes #7511 

> Is there anything in the PR that needs further explanation?

Based on #7417

### CODE REVIEW

**Comment by emmacharp** [comment]
@ybiquitous Looks like I missed the 16.3.0 by a day... heheh.

First time contributing. Hope I did everything right.

File: lib/rules/rule-selector-property-disallowed-list/README.md [hunk 1]
```diff
@@ -61,3 +61,27 @@ a { padding-top: 0px; }
 ```css
 html[data-foo] { color: red; }
 ```
+
```
Comment (ybiquitous) [inline lib/rules/rule-selector-property-disallowed-list/README.md:64]: [suggest] Let's add a new heading.

```suggestion

## Optional secondary options

```

File: lib/rules/rule-selector-property-disallowed-list/README.md [hunk 2]
```diff
@@ -61,3 +61,27 @@ a { padding-top: 0px; }
...
+### `ignore: ["keyframe-selectors"]`
+
+Ignore keyframe selectors.
+
+Given:
+
+```json
+[{"/^[a-z]+$/": ["opacity"]}, { "ignore": ["keyframe-selectors"] ]
```
Comment (ybiquitous) [inline lib/rules/rule-selector-property-disallowed-list/README.md:72]: [suggest] Let's fix the broken JSON.

```suggestion
[{"/^[a-z]+$/": ["opacity"]}, {"ignore": ["keyframe-selectors"]}]
```

File: lib/rules/rule-selector-property-disallowed-list/README.md [hunk 3]
```diff
@@ -61,3 +61,27 @@ a { padding-top: 0px; }
...
+
+<!-- prettier-ignore -->
+```css
+@keyframes fade-in {
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
```
Comment (ybiquitous) [inline lib/rules/rule-selector-property-disallowed-list/README.md:85]: [suggest] Super-minor. We may save a few lines, although you can ignore this comment.

```suggestion
  from { opacity: 0; }
  to { opacity: 1; }
```

File: lib/rules/rule-selector-property-disallowed-list/index.mjs [hunk 4]
```diff
@@ -17,24 +19,46 @@ const meta = {
...
+			ruleName,
+			{
+				actual: primary,
+				possible: [validateObjectWithArrayProps(isString, isRegExp)],
+			},
+			{
+				actual: secondaryOptions,
+				possible: {
+					ignore: ['keyframe-selectors'],
+					splitList: [isBoolean],
```
Comment (ybiquitous) [inline lib/rules/rule-selector-property-disallowed-list/index.mjs:35]: [suggest] Let's remove `splitList`, which is no added actually.

```suggestion
```

**Review by ybiquitous — CHANGES_REQUESTED** [review]
@emmacharp Thank your for your first-time contribution!

I've left some suggestions, but overall good 👍🏼

File: lib/rules/rule-selector-property-disallowed-list/index.mjs (hunk 4 above)
Comment (emmacharp) [reply lib/rules/rule-selector-property-disallowed-list/index.mjs:35]: Bad copy & paste. Shameful! Heheh.

**Comment by emmacharp** [comment]
I'll fix it all at some time today.

**Comment by emmacharp** [comment]
Done!

**Comment by ybiquitous** [comment]
Thanks. Can you build `*.cjs` files and push them?

**Comment by emmacharp** [comment]
Oups. I'll get used to it. Hehe.

**Review by ybiquitous — APPROVED** [review]
Thank you. LGTM 👍🏼

**Pull request merged**